hal/lib/athena/ChipObject.h

```diff
@@ -13,12 +13,6 @@
 #include <stdint.h>
 
 #include "FRC_FPGA_ChipObject/RoboRIO_FRC_ChipObject_Aliases.h"
```

Comment:
I moved them to the top as NOLINT headers and filed issue #247. It should probably be tagged as "bug in dependency".